docs/jumpstart/week-9.md

```diff
@@ -0,0 +1,343 @@
+
+## Distinct
+Normally, we can use the `SELECT` keyword to select all relevant rows from a table. But what if
+ we want to only select distinct rows? That is, we want to select all relevant rows that are not
+  identical (by the given criteria).
+  
+For example, let's say we wanted a list of all distinct product names from our `products` table
+ defined above. Rather than query all products and remove duplicates, we can use SQL to query
+  only distinct values:
+  
+```sql
+SELECT DISTINCT name FROM products
+```
+
+We can also apply `DISTINCT` to multiple columns. This works how you would think: 
+
+```sql
+SELECT DISTINCT name, price FROM products
+```
+
+selects all rows whose combination of name and price is unique. In other words, the resulting
+ table will have two columns, `name` and `price`, and no two rows will have the same value for both.
+
+## Union
+Some of you may already be familiar with the mathematical notion of a union -- the combination of
+ two sets. Similarly, we can use SQL's `UNION` keyword to combine the results of two queries
+ . Note that the queries must reference the same columns: a union of two sets of rows with different
+  columns doesn't make sense.
+  
+Like union in math, `UNION` does not include any duplicates (unless you use `UNION ALL`).
+
+Here's an example of a union query:
+
+```sql
+SELECT column1, column2 FROM table WHERE column1 > 50
+UNION
+SELECT column1, column2 FROM table WHERE column1 < 5
+```
+
+## Aggregate Functions
+If you've used Excel before, you probably appreciate the statistics operations that allow you to
+ quickly get info about data entries, like average() and max(). As you might expect, Postgres
+  includes this functionality out of the box.
+
+### Count
+For example, let's say we want to know how many entries are in a table. This is trivial:
+
+```sql
+SELECT COUNT(*) FROM table
+```
+
+We can add `WHERE` clauses to count only entries that satisfy some condition, like this:
+
+```sql
+SELECT COUNT(*) FROM table WHERE column3 = false
+```
+
+This query returns a table with only a single column `count` and one entry containing the count.
+
+### Min/Max
+To get the minimum or maximum values of a numeric column in a table, we can use the `MIN()` and `MAX
+()` functions, like this:
+
+```sql
+SELECT MIN(price) FROM products
+```
+
+Like `COUNT`, this returns a table with a single column `min` and one entry containing the
+ minimum value of the queried column.
+ 
+### Avg
+Average returns the average of a numeric column:
+
+```sql
+SELECT AVG(price) FROM products
+```
+
+### Sum
+Sum returns the sum of a numeric column:
+
+```sql
+SELECT SUM(price) FROM products
+```
+
+## Group By & Having
```

Comment:
I think it may be beneficial to add an example of where 'WHERE' and 'HAVING' are both being used, to explain how the 'HAVING' clause is applied to the rows in the result set after the 'WHERE' clause is applied.